File: src/app/src/main/java/servicestack/net/kotlinlinqexamples/support/Product.java
package servicestack.net.kotlinlinqexamples.support;

/**
 * Created by mythz on 7/26/2015.
 */
public class Product {
    public Integer productId;
    public String productName;
    public String category;
    public Double unitPrice;
    public Integer unitsInStock;

    public Product(Integer productId, String productName, String category, Double unitPrice, Integer unitsInStock) {
        this.productId = productId;
        this.productName = productName;
        this.category = category;
        this.unitPrice = unitPrice;
        this.unitsInStock = unitsInStock;
    }

    @Override
    public String toString() {
        return "(Product " +
                "id=" + productId +
                ", name=" + productName +
                ", cat=" + category +
                ", price=" + unitPrice +
                ", inStock=" + unitsInStock +
                ')';
    }
}

File: src/app/src/main/java/servicestack/net/kotlinlinqexamples/support/Product.kt
package servicestack.net.kotlinlinqexamples.support

class Product(var productId: Int, var productName: String, var category: String, var unitPrice: Double, var unitsInStock: Int) {

    override fun toString(): String {
        return "(Product id=$productId, name=$productName, cat=$category, price=$unitPrice, inStock=$unitsInStock)"
    }
}
